apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.html

```diff
@@ -1523,6 +1521,7 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                                                 class="form-control"
                                                 [ngModel]="vm.selectedLedger.voucherNo"
                                                 [disabled]="true"
+                                                [style.width.px]="80"
```

Comment:
please remove this, we have removed this from test, other PR are not merged yet, therefore this line is there on stage

Reply:
Done